packages/react/src/themes/teams/components/Dialog/dialogVariables.ts

```diff
@@ -0,0 +1,40 @@
+import { DialogVariables as BaseDialogVariables } from '../../../base/components/Dialog/dialogVariables'
+import { pxToRem } from '../../../../lib'
+
+export interface DialogVariables extends BaseDialogVariables {
```

Comment:
As this extends `BaseDialogVariables` it should contain only newly added vars, not the ones inherited from Base:
```jsx
  boxShadow: string
  foregroundColor: string
  headerFontSize: string
  headerFontWeight: number
```